pyproject.toml

```diff
@@ -38,10 +38,11 @@ dependencies = [
 "datasets>=2.15.0",
 "fire",
 "simpleeval",
+"packaging",
 ]
 
 [project.optional-dependencies]
-dev = ["wheel", "packaging", "ninja", "scikit-learn>=1.0, <2.0", "boto3"]
```

Comment:
Is packaging not only used in the building of the wheel and thus is needed as a required dependency?

Reply:
It's a dependency of the accelerate lib. It should be installed automatically with the lib. So I guess this change is not required. let me check...